scripts/superdesk-authoring/authoring.js

```diff
@@ -1662,9 +1662,21 @@
                 scope.destination_last = null;
                 var PREFERENCE_KEY = 'destination:active';
 
+                scope.modelTimeFormat = 'HH:mm:ss';
+                scope.viewTimeFormat = config.view.timeformat;
+
                 scope.$watch('item', activateItem);
                 scope.$watch(send.getConfig, activateConfig);
 
+                scope.$watch('item.embargo_date', function(newVal, oldVal) {
```

Comment:
There is already a watch for here: https://github.com/superdesk/superdesk-client-core/blob/master/scripts/superdesk-authoring/metadata/metadata.js#L134